codes/csharp/hello-algo.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32421.90
```

Comment:
Could we use a more lightweight IDE to run the code, like VSCode? And is it possible to make every source code file independent? That is, the users could copy just one file and run it, not the whole directory.

Reply:
I will delete this file and the project can be run in every editor or ide.But for C#, running a single file is unusual, and for systems other than Win that must rely on other tools.

Reply:
That's OK. I‘m not familiar with C# (I regard Java as an alternative to C#, but I forget the multi-platform capability of the two languages has a large gap.)

Do you know a solution to make the C# code run on all common systems like Win, macOS, and ubuntu(Linux)? Cause the users could use different systems. Hope for your solution, thanks!

Reply:
But for C#, running a single file is unusual

Hmmm, but I believe it is OK for learning leetcode and algorithm. Click one button running the code is the wish of us and this book.

Reply:
The dotnet project supports multiple platforms, but compiling a single *.cs file can only be compiled into an exe.
When I found that I could not compile and run a single file on a mac which is why I created the project. The current project can run on any system with a .net environment.

I looked for information to find other systems can be installed https://www.mono-project.com/ to run *.cs files 
compiled exe, but I feel that this is a little weird.

C# is very similar to java, so it's very easy for me to translate java code (basically ctrl+c)

Reply:
I see. Maybe this does be the best solution.